client/src/js/viruses/components/Detail/Detail.js

```diff
@@ -10,6 +10,7 @@ import IsolateEditor from "./Editor";
 import EditVirus from "./EditVirus";
 import General from "./General";
 import History from "./History";
+import Schema from "./Schema";
```

Comment:
Save the schema code for another PR. Your bug fixes will be released sooner in a separate release from the schema feature.